adoc/admin-security-ldap.adoc

```diff
@@ -4,6 +4,124 @@
 
 {productname} supports user authentication via external LDAP server like 389 Directory Server and Active Directory by updating the builtin Dex LDAP connector configuration.
 
+=== Deploying an external 389 Directory Server with an automatically generated self-signed certificate
+
+The 389 Directory Server image `registry.suse.com/caasp/v4/389-ds:1.4.0` will automatically generate a self-signed certificate and key. 389 Directory Server can be deployed with customized configuration and container command:
+
+. Prepare customized 389 Directory configuration to enter in Terminal:
++
+----
+INSTANCE_NAME=localhost                         # Directory Server Name
+ROOT_DN="cn=admin,dc=example,dc=org"            # Admin Account
+ROOT_PASSWORD=                                  # Admin Password
+SUFFIX="dc=example,dc=org"                      # Domain Suffix
+DATA_DIR=<PWD>/389_ds_data                      # Directory Server Data on Host Machine to Mount
+----
+
+. Execute `docker` command to deploy 389 Directory Server in same Terminal. This will start a non-TLS port (389) and a TLS port (636) together with an automatically self-signed certificate and key.
++
+----
+docker run -d \
+	-p 389:389 \
+	-p 636:636 \
+	-e INSTANCE_NAME=<INSTANCE_NAME> \
+	-e ROOT_DN=<ROOT_DN> \
+	-e ROOT_PASSWORD=<ROOT_PASSWORD> \
+	-e SUFFIX=<SUFFIX> \
+	-v <DATA_DIR>:/data \
+	--name 389-ds registry.suse.com/caasp/v4/389-ds:1.4.0
+----
+
+=== Deploying a 389 Directory Server with an external certificate
+
+To replace the certificate with your own, follow these steps:
+
+. Stop running container:
++
+----
+docker stop 389-ds
+----
+
+. Copy external certificate you would like to use `<rootCA.pem>` to a mounted data directory `<DATA_DIR>/config/`
+
+. Run bash in entrypoint to access container with mounted data directory from previous step:
++
+----
+docker run --rm -it \
+  -p 389:389 \
+  -p 636:636 \
+  -v <DATA_DIR>:/data \
+  --entrypoint /bin/bash \
+  --name 389-ds registry.suse.com/caasp/v4/389-ds:1.4.0
+----
+
+. List certificates in `/etc/dirsrv/slapd-<INSTANCE_NAME>`
++
+----
+certutil -L -d /etc/dirsrv/slapd-<INSTANCE_NAME> -f pwdfile.txt
```

Comment:
User needs to be in instance directory to access pwdfile.txt. Suggest to use full path.